app/bundles/CampaignBundle/Entity/Campaign.php

```diff
@@ -141,7 +141,7 @@ public static function loadMetadata(ORM\ClassMetadata $metadata)
             ->nullable()
             ->build();
 
-        $builder->addNamedField('allowRestart', 'integer', 'allow_restart');
+        $builder->addNamedField('allowRestart', 'boolean', 'allow_restart');
```

Comment:
Resolved in commit 1b8de9f7cd6486b520c8eb8b4e141e17b7498267

Reply:
I think it could be a compromise. The `allowRestart` property would be type of `int` but the getter and setter would work with `bool` as it was so far. We can retype int to bool and back.

Reply:
Fixed in commit c6ea82219dc0bb654e719c01c04ef768eb9054cc